core/store/migrations/migration1611847145/migrate.go

```diff
@@ -0,0 +1,21 @@
+package migration1611847145
+
+import "github.com/jinzhu/gorm"
+
+// Migrate changes trigger to only notify on unfinished runs
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+	CREATE OR REPLACE FUNCTION notifyPipelineRunStarted() RETURNS TRIGGER AS $_$
```

Comment:
Seems simpler just to not start the goroutines that process these notifs?

Reply:
What was the resolution here?

Reply:
This function is fine